cliquet/resource/__init__.py

```diff
@@ -240,6 +248,11 @@ def collection_get(self):
             next_page = self._next_page_url(sorting, limit, lastrecord, offset)
             headers['Next-Page'] = encode_header(next_page)
 
+        records = [
+            {field: val for field, val in record.items() if field in fields}
```

Comment:
> What if a known field is not present in the record ?

I don't undestand. `fields` is a subset of _get_known_fields. I assume that all fields of a record are in `known_fields` then I filter the fields by `?_filter=`. What could be wrong? What do you mean?

I agree that no filtering should be made if there is no `?_fields=`. I will also rename `fields` with `partial_fields`.